hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -822,26 +849,26 @@ protected void testRenameDirToFile() throws Exception {
     ContractTestUtils.touch(fs, file1Destin);
     Path abcRootPath = new Path(fs.getUri().toString() + "/a/b/c");
     fs.mkdirs(abcRootPath);
-    try {
-      fs.rename(abcRootPath, file1Destin);
-      Assert.fail("key already exists /root_dir/file1");
-    } catch (FileAlreadyExistsException faee) {
-      // expected
-    }
+    Assert.assertFalse("key already exists /root_dir/file1",
+            fs.rename(abcRootPath, file1Destin));
   }
```

Comment:
Above rename related change already contained in PR #1607.

Reply:
@linyiqun  yes, you are right. Since that is in progress, I just added to make clean build report. We can push #1607 push/upstream it first and then I will rebase this PR on top of that. Hope that would be fine.